src/connectors/index.js

```diff
@@ -0,0 +1,32 @@
+import jwt from 'jsonwebtoken'
+
+import { createDeleteConnector } from 'standard-json-api-connectors'
+import { ValidationError } from 'standard-api-errors'
+
+const secrets = process.env.SECRETS.split(' ')
+const emailFoxApiUrl = process.env.EMAILFOX_API_URL
```

Comment:
It should have a generic name, since we will use this accounts api for many different products.